src/middlewared/middlewared/etc_files/local/ssh/sshd_config.mako

```diff
@@ -108,3 +108,7 @@ ChallengeResponseAuthentication yes
 PrintMotd no
 % endif
 SetEnv LC_ALL=C.UTF-8
+% if not ssh_config['adminlogin']:
+Match User admin
```

Comment:
Hmm, cant the admin user be called anything?

Reply:
@william-gr no, for the sake of simplicity it can't. In 23.10 we'll have per-user/group option, but here we'll only replicate root functionality for the built-in `admin` user we allow to create using 22.12 CD installer.